packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3912,7 +3912,12 @@ class Flex extends MultiChildRenderObjectWidget {
   /// `start` and `end` in the horizontal direction.
   ///
   /// Defaults to the ambient [Directionality].
-  ///
+  /// 
+  /// If `textDirection` is [TextDirection.rtl], then the direction in which 
+  /// text flows starts from right to left. Otherwise, if `textDirection` is 
+  /// [TextDirection.ltr], then the direction in which text flows starts from 
+  /// left to right.
+  /// 
```

Comment:
```suggestion
  ///
  /// If `textDirection` is [TextDirection.rtl], then the direction in which
  /// text flows starts from right to left. Otherwise, if `textDirection` is
  /// [TextDirection.ltr], then the direction in which text flows starts from
  /// left to right.
  ///
```

Reply:
Will fix it.....Thank you..

Reply:
Although this is a correct statement about `textDirection` in general, it's not really help for understanding how the `Row` lays out its children.  If you rephrased the point (in terms of the row's children) and explained the implications of the start and end parameters, that would help.

#58019 was really a request to enhance the `Row` _class_ API doc, by mentioning the support for RTL layout a little earlier.

Reply:
Sure. @HansMuller